xml/System.Windows.Forms/GiveFeedbackEventArgs.xml

```diff
@@ -289,9 +295,12 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value indicating whether a layered window drag image is used.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> for <see cref="P:System.Windows.Forms.GiveFeedbackEventArgs.UseDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />.</value>
```

Reply:
👍